src/emulator.ts

```diff
@@ -221,7 +220,7 @@ export class EmacsEmulator implements IEmacsController, vscode.Disposable {
   }
 
   public onDidChangeTextEditorSelection(e: vscode.TextEditorSelectionChangeEvent): void {
-    if (new EditorIdentity(e.textEditor).isEqual(new EditorIdentity(this.textEditor))) {
+    if (e.textEditor.document.uri === this.textEditor.document.uri) {
```

Comment:
This line became the same as https://github.com/whitphx/vscode-emacs-mcx/pull/1455/files#diff-eb9908a6fc5cf5cc98a6bedc6fd086107e314955f31251552c05c49e769b925aR207
-> Is comparison of `Uri` objects correct like this? Can the `Uri` objects be compared with their identities?

Reply:
I could not find a clear answer, but we can find many places where `uri.toString()` are compared at the [vscode repo](https://github.com/microsoft/vscode) such as 
https://github.com/microsoft/vscode/blob/e3cc677d15dc7a2bb9b2da6332c049361481c85a/src/vs/workbench/contrib/notebook/browser/notebook.contribution.ts#L302

So we should use `uri.toString()` here too.